common/src/main/resources/languages/en-gb.yml

```diff
@@ -39,4 +39,7 @@ error_console_switch_channels: 'Error: You cannot switch to another channel from
 error_group_messages_disabled: '[Error:](#ff3300) [You cannot send messages to multiple people.](#ff7e5e)'
 error_group_messages_max: '[Error:](#ff3300) [You can only send messages to a maximum of %1% people at once.](#ff7e5e)'
 error_players_not_found: '[Error:](#ff3300) [Could not find any of the specified players; are they online?](#ff7e5e)'
-error_reply_none_online: '[Error:](#ff3300) [Nobody in the last group of people you messaged are still online.](#ff7e5e)'
\ No newline at end of file
+error_reply_none_online: '[Error:](#ff3300) [Nobody in the last group of people you messaged are still online.](#ff7e5e)'
+error_last_message_not_group: '[Error:](#ff3300) [The last message you received was not a group message.](#ff7e5e)'
+error_no_messages_opt_out: '[Error:](#ff3300) [You have not yet sent or received messages.](#ff7e5e)'
+removed_from_group_message: 'Removed you from the group message between: %1%'
```

Comment:
Since this isn't a console message, we should format it using the message color scheme - (#00fb9a)
Something like `[Removed you from the group message between](#00fb9a)`

------
I'd also like your thoughts on the formatting of the player list here.
In most cases, group messages will be between 3-5 people - so having it display "Removed you from the group message between: William, Player" looks a little stiff. Typically, a bit of code to replace the last comma separator with an *and*/*&* would be good to make this display a little nicer - though it might be worth considering that this may not work great with other languages, etc.

Reply:
I guess I could add an option for setting the conjunction in the language files, that would probably be the easiest solution for this. I can think of a few other solutions, but those would be hard to implement and require very specific code changes.

Reply:
An option sounds reasonable to me :)